### ISSUE TITLE
some question about simulation

### ISSUE THREAD

**Issue by kangliyu1** [suisuisi/FPGAandCNN#1]
Hello! Why are all my simulation results identified as "a", has your simulation passed? Can you give me some pointers? Many thanks!

**Comment by zhanyan110** [comment]
Yes, my simulation result is also "a" , do you find where is the issue possible ?

**Comment by zhanyan110** [comment]
> Hello! Why are all my simulation results identified as "a", has your simulation passed? Can you give me some pointers? Many thanks!

Do you fix this issue ?

**Comment by kangliyu1** [comment]
> > Hello! Why are all my simulation results identified as "a", has your simulation passed? Can you give me some pointers? Many thanks!
> 
> Do you fix this issue ?

The author of the source of your open source project has fixed this bug, you can go to download it again
